oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -215,6 +223,14 @@ Frontend Technology Selection
 
    You can find out more about Sass in the official `Sass documentation`_.
 
+.. _use-css-variables:
```

Comment:
Using this cross-reference causes the numbers to start over. I propose we don't use this cross-reference and just say in the above note to look at the next item.